tasks/main.yml

```diff
@@ -5,6 +5,10 @@
     name: unzip
     state: present
 
+- name: Ensure at least 1 locale exists
+  community.general.locale_gen:
+    name: 'en_US.UTF-8'
+
```

Comment:
This was removed later in the same PR, so neither of these belongs in this PR

Reply:
Looks like you squashed all commits into one, which doesn't leave a nice trail of what & why things changed. I'd be happy to see the original commit history, except for those unnecessary ones and with a clearer phrasing in the commit subject and body.